.travis.yml

```diff
@@ -109,8 +109,3 @@ matrix:
     - rvm: 2.3.0
       env: RAILS_VERSION=master
   fast_finish: true
-
-branches:
-  only:
-    - master
-    - /^\d+-\d+-maintenance$/
```

Comment:
This should be left in, Travis triggers builds in two ways, by pushes to branches, and by pushing to PRs, without this we build twice on those PRs